idseq_dag/steps/generate_alignment_viz.py

```diff
@@ -302,61 +282,6 @@ def get_sequences_by_accession_list_from_file(accession2seq, nt_loc_dict,
                 accession_info['ref_seq_len'] = len(ref_seq)
                 accession_info['name'] = seq_name
 
-    @staticmethod
```

Comment:
Good catch by @MarkAZhang -- the phylotree still fetches references from s3.   Fortunately it only fetches a few, not tons and tons like the alignment viz.   So, we'll have to keep this code for phylo, just dial down the parallelism.  Still a win not to use it from alignment viz, but will take one more iteration of this PR + testing.